src/isa/exec.rs

```diff
@@ -698,7 +689,6 @@ impl InstructionSet for BytesOp {
                 };
                 f().unwrap_or_else(|| {
                     regs.st0 = false;
-                    regs.set(RegA::A16, Reg32::Reg0, MaybeNumber::none());
```

Comment:
Why are we removing this if the spec below says
```
If the first or the second string is `None`, sets `st0` to `false` and `a16[0]` to `None`.
```

Reply:
Yet still the `Rev` must reset the destination to the uninitialized state:
```
/// If the source string register is uninitialized, resets destination to the uninitialized
```

So I assume the above line should stay and be changed to
```suggestion
                    regs.s16[reg2.as_usize()] = None;
```

Reply:
true